src/app/page.tsx

```diff
@@ -1,113 +1,125 @@
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Templates{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Explore the Next.js 13 playground.
-          </p>
-        </a>
+      if (data) {
+        setResult(data.example);
+        console.log("payload: ", payload, "data: ", data);
+      }
+      if (error) {
+        setErrorMessage(error.message);
+        console.log(error);
+      }
+    }
+  }
+
+  function copyResult() {
+    navigator.clipboard.writeText(result ? result : "");
+    setIsClicked(true);
+    setTimeout(() => {
+      setIsClicked(false);
+    }, 500);
+  }
 
-        <a
-          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel="noopener noreferrer"
+  return (
+    <main className="flex min-h-screen flex-col items-start space-y-10 px-24 py-16">
+      <div className="flex flex-row justify-between items-center w-full">
+        <h1 className="text-4xl">
+          <span className="text-green-400 mr-4">{"{ }"}</span>faker
+        </h1>
+        <Link
+          href={"https://github.com/debbyalmadea/faker"}
+          className="px-4 py-2 bg-transparent text-neutral-300 border border-neutral-300 hover:bg-neutral-950 active:bg-neutral-900 rounded-md"
         >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Deploy{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
+          <FontAwesomeIcon icon={faGithub} />
+          <span className="ml-4">See Project</span>
+        </Link>
       </div>
+      <div className="items-center w-full justify-between font-mono text-sm border-b px-8 rounded-lg bg-gradient-to-b pb-6 pt-8 backdrop-blur-3xl border-neutral-800 bg-zinc-800/30 from-inherit lg:bg-gray-200 lg:bg-zinc-800/30">
+        <div className="flex flex-row space-x-3">
+          <div className="bg-red-400 w-3 h-3 rounded-full" />
+          <div className="bg-yellow-400 w-3 h-3 rounded-full" />
+          <div className="bg-green-400 w-3 h-3 rounded-full" />
+        </div>
+        <textarea
```

Comment:
Add example? Ideally user should know that they can put the type structure in their respective language to use this.
If we only show the typescript example, user might think that it only support ts, how do you think we should show this example? @debbyalmadea

Reply:
Added example for typescript, javascript, python, and golang in the newest commit